### PR TITLE
Added Meraki MR36 and MR44

### PR DESCRIPTION
**Pull request by vittore**

### CODE REVIEW

**Comment by vittore** [comment]
Please add

File: device-types/Cisco/Meraki-MR57.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@
 manufacturer: Cisco
 model: MR57
 slug: MR57
-u_height: 0
```
Comment (danner26) [inline device-types/Cisco/Meraki-MR57.yaml:5]: These are not rack mountable to my knowledge. We should leave `u_height: 0` defined

File: device-types/Cisco/Meraki-MR56.yaml [hunk 2]
```diff
@@ -2,7 +2,6 @@
 manufacturer: Cisco
 model: MR56
 slug: MR56
-u_height: 0
```
Comment (danner26) [inline device-types/Cisco/Meraki-MR56.yaml:5]: These are not rack mountable to my knowledge. We should leave `u_height: 0` defined

File: device-types/Cisco/Meraki-MR53E.yaml [hunk 3]
```diff
@@ -2,7 +2,6 @@
 manufacturer: Cisco
 model: MR53E
 slug: MR53E
-u_height: 0
```
Comment (danner26) [inline device-types/Cisco/Meraki-MR53E.yaml:5]: These are not rack mountable to my knowledge. We should leave `u_height: 0` defined

File: device-types/Cisco/Meraki-MR53.yaml [hunk 4]
```diff
@@ -2,7 +2,6 @@
 manufacturer: Cisco
 model: MR53
 slug: MR53
-u_height: 0
```
Comment (danner26) [inline device-types/Cisco/Meraki-MR53.yaml:5]: These are not rack mountable to my knowledge. We should leave `u_height: 0` defined

**Review by danner26 — CHANGES_REQUESTED** [review]
These are not rack mountable to my knowledge. We should leave `u_height: 0` defined

**Comment by danner26** [comment]
@vittore I also noticed your two additions are missing a newline at the end of the file. Please add a newline so validation succeeds. Thank you

**Comment by vittore** [comment]
Done.

Thanks.
v.


Il giorno mar 27 dic 2022 alle ore 17:11 Daniel Anner <
***@***.***> ha scritto:

**Review by danner26 — APPROVED** [review]

**Comment by danner26** [comment]
Looks good! Thanks @vittore 
I will merge this soon

**Pull request merged**